build/common.sh

```diff
@@ -682,10 +682,14 @@ function kube::build::sync_to_container() {
   # they will not be re-generated by 'make'.
   kube::build::rsync \
     --delete \
+    --filter='+ /staging/**' \
```

Comment:
My bad, my comment wasn't totally clear. This line shouldn't have been restored.. just the generated ones below. Sorry!